osdk/src/commands/new/kernel.template

```diff
@@ -0,0 +1,22 @@
+#![no_std]
+#![no_main]
+#![forbid(unsafe_code)]
+
+#[macro_use]
+extern crate ktest;
+
+use aster_frame::prelude::*;
+
+#[aster_main]
+fn kernel_main() -> ! {
+    println!("Hello world from guest kernel!");
+    loop {}
```

Comment:
Can we avoid using `loop {}` here? For example, using Qemu::exit(Exitcode::Success) may be better.

Reply:
Qemu exit depends on an additional debug exit device. I suggest adding a proper shutdown facility in the framework.